### ISSUE TITLE
Respect v-model modifier lazy

### ISSUE THREAD

**Issue by gebeer** [mattzollinhofer/vue-typeahead-bootstrap#114]
Thanks for maintaining this.
I need lazy behaviour on v-model, like `v-model.lazy="name"`. But it doesn't seem to be implemented in this component.

My problem: the component is bound to a data property 'name'. On change of this property, I trigger a filter method that filters through hundreds of objects. ATM my filter method is being triggered on every keystroke which is bad for performance.

Would be great if you could implement the native .lazy modifier on v-model.

**Comment by mattzollinhofer** [comment]
Yeah, I can see the value in this idea. Honestly, there’s probably a lot of other things that need work first, but I’ll keep this on the list and will be interested if others see the need as well. 

A workaround for you might be to use denounce to limit calls while typing, but obviously I don’t know hire you’re implementing this so that suggestion might not work.

**Comment by gebeer** [comment]
Great to hear that you'll keep this in the loop.
Meanwhile I managed to workaround using your custom hit event which does exactly what I need.
```
<vue-typeahead-bootstrap
          @hit="setName"
          ...
```
then in my methods
```
  methods: {
    setName(val) {
      this.name = val;
    },
  },
```
Now I can watch the name property and filter my results accordingly :-)
